assets/docs/configuration/targets/http-full-example.hcl

```diff
@@ -17,6 +17,9 @@ target {
     # Request timeout in seconds (default: 5)
     request_timeout_in_seconds = 2
 
+    # Request timeout in milliseconds (default: 5000)
+    request_timeout            = 2000
```

Comment:
Just `request_timeout` or add `...in_millis` suffix...?

Reply:
While both exist, let's be explicit. We may change it when we make a breaking change later.